habitLoggerApp.Natephor/UserInterface.cs

```diff
@@ -0,0 +1,71 @@
+namespace HabitLoggerApp;
+
+public class UserInterface
+{
+    public string GetUserInput(string message)
+    {
+        string? userInput = null;
+        while (string.IsNullOrEmpty(userInput))
+        {
+            Console.WriteLine(message);
+            userInput = Console.ReadLine();
+        }
+
+        return userInput;
+    }
+
+    public int GetUserNumberInput(string message)
```

Comment:
🟠 **Validation Improvements**

💡 Can you do negative quantities? Apparently you can 😄 !

![image](https://github.com/user-attachments/assets/116dfd2b-156c-42f3-b29e-b42a6cff5904)